boa/src/exec/mod.rs

```diff
@@ -591,7 +614,7 @@ impl Interpreter {
                                     .environment
                                     .initialize_binding(name, expr.clone());
                             }
-                            Node::UnaryOp(UnaryOp::Spread, ref expr) => {
+                            Node::Spread(ref expr) => {
```

Comment:
Why was this moved out of `UnaryOp`?

Reply:
In the standard/specification `Spread` is not a part of the unary expression.
see: https://tc39.es/ecma262/#prod-UnaryExpression